networkx/algorithms/euler.py

```diff
@@ -15,10 +21,15 @@
 
 from ..utils import arbitrary_element
 
+from collections import deque
+
 __author__ = """\n""".join(['Nima Mohammadi (nima.irt[AT]gmail.com)',
-                            'Aric Hagberg <hagberg@lanl.gov>'])
+                            'Aric Hagberg <hagberg@lanl.gov>',
```

Comment:
This is off topic from this PR but...
I'd like to keep the barrier as low as possible for contributing.

There is no networkx organization at this point.  If we wanted to consider a copyright license agreement we'd have to create something like that first. 

The current situation is as @jfinkels describes.   We accept code contributions from authors who must have the authority and ability to distribute the code under the modified BSD license.   We don't require a copyright statement if the author does not provide one and are willing to add the standard NetworkX copyright if requested.   I'd prefer not to include any code without an explicit copyright statement.

We have no clear license on the documentation other than BSD (which isn't quite appropriate for documentation).